website/docs/user_guide/environments.md

```diff
@@ -56,6 +56,14 @@ In order for the SDK to download the feature toggle configuration for the correc
 
 ![An API key creation form. The form's fields are "username", "token type", "project", and, crucially, "environment". The development environment is selected.](/img/environments_api_keys.png 'Create Environment specific API Keys')
 
+## Cloning environments
+
+If you have a lot of feature toggles, it can be tedious to configure the same activation strategies for each environment. To make this easier, you can clone an environment. This will copy all feature toggle configurations and activation strategies from the source environment to the target environment.
+
+This can also prove to be a safe way to experiment with activation strategies without affecting production-facing environments. You can clone the production environment, and then test out the new activation strategies in the cloned environment. If everything works as expected, you can then apply the same activation strategies to the production environment.
+
+When cloning an environment, you can choose several options to customize your new environment: name, type of environment, which projects to clone the environment configuration in, and whether you wish to keep the same user permissions. You'll also be provided the choice of generating a new API key bound to the newly cloned environment right away, or generating one yourself at a later stage.
+
```

Comment:
We should add a paragraph here explaining how access control works for the newly created enviroments, as @sighphyre mentioned in relation to the how-to guide. This is where we should explain it.